TCAT/Controllers/RouteDetail+DrawerViewController.swift

```diff
@@ -40,7 +40,7 @@ enum RouteDetailItem {
 class RouteDetailDrawerViewController: UIViewController {
 
     var safeAreaCover: UIView?
-    var summaryView = SummaryView()
+    var summaryView: SummaryView!
```

Comment:
make private?

Reply:
I'm going to tackle the private vars in a different PR hopefully.